### ISSUE TITLE
Support a kartridge runner/ support importing games from there

### ISSUE THREAD

**Issue by romulasry** [lutris/lutris#1956]
https://www.kartridge.com/games

**Comment by AccountOneOff** [comment]
It needs to meet the following for that to happen:

- A way to launch games.
- A way to install games.
- A way to uninstall games.

If you can find how to do that via cli then a runner can be written. See the [Uplay](https://github.com/lutris/lutris/issues/911) and [Origin](https://github.com/lutris/lutris/issues/808) runner issues.

**Comment by strycore** [comment]
There are 2 sort of things we can add support for, runners and services. Runners are tools that can launch games by themselves (like Uplay, Origin, Battle.net, Steam). Services are for importing games into the library but the games themselves aren't tied to a particular runner. We have services like local desktop games, GOG, also Steam (for syncing installed games) and in the future Humble Bundle and TOSEC based ROM import.
If Kartridge is a client that can install and launch games then it's a runner, if you can download games and run them with whatever then it's a service. Services still require an API that can list games for an user and download game files.

**Comment by AccountOneOff** [comment]
It's a runner.

![Screenshot from 2019-03-27 16-24-33](https://user-images.githubusercontent.com/15083452/55118801-d2a2ea80-50ac-11e9-9991-d2081d950974.png)

**Comment by strycore** [comment]
yeah, usually, a good workaround is to have an installer for that game client until the runner is implemented.
